<!DOCTYPE html>
<html lang="en">
  <div>
    <h1>GigaDB Dataset Simplified View</h1>
    <h3>
      Click <a href="http://doi.org/10.5524/100397">here</a> to return to full
      HTML view.
    </h3>
    <br />
  </div>
  <div>
    <table class="datasetTable" width="95%" align="center">
      <tbody>
        <tr>
          <td>DOI</td>
          <td>
            <a
              href="http://doi.org/10.5524/100397"
              target="_blank"
              rel="noopener"
              >10.5524/100397</a
            >
          </td>
        </tr>
        <tr>
          <td>Release_Date</td>
          <td>December 22, 2017</td>
        </tr>
        <tr>
          <td>Title</td>
          <td>
            Supporting data for "Finding Nemo: Hybrid assembly with Oxford
            Nanopore and Illumina reads greatly improves the Clownfish
            (Amphiprion ocellaris) genome assembly"
          </td>
        </tr>
        <tr>
          <td>Citation</td>
          <td>
            Tan, M, H; Austin, C, M; Hammer, M, P; Lee, Y, P; Croft, L, J; Gan,
            H, M (2017): Supporting data for "Finding Nemo: Hybrid assembly with
            Oxford Nanopore and Illumina reads greatly improves the Clownfish
            (Amphiprion ocellaris) genome assembly" GigaScience Database.
            <a
              href="http://doi.org/10.5524/100397"
              target="_blank"
              rel="noopener"
              >http://dx.doi.org/10.5524/100397</a
            >
          </td>
        </tr>
        <tr>
          <td>Data_Type</td>
          <td>Genomic</td>
        </tr>
        <tr>
          <td>Data_Type</td>
          <td>Transcriptomic</td>
        </tr>
        <tr>
          <td>Keyword</td>
          <td>clownfish</td>
        </tr>
        <tr>
          <td>Keyword</td>
          <td>long reads</td>
        </tr>
        <tr>
          <td>Keyword</td>
          <td>genome</td>
        </tr>
        <tr>
          <td>Keyword</td>
          <td>transcriptome</td>
        </tr>
        <tr>
          <td>Keyword</td>
          <td>hybrid assembly</td>
        </tr>
        <tr>
          <td>Dataset_Summary</td>
          <td>
            Some of the most widely recognised coral-reef fishes are clownfish
            or anemonefishes, members of the family Pomacentridae (subfamily:
            Amphiprioninae). They are popular aquarium species due to their
            bright colours, adaptability to captivity and fascinating behavior.
            Their breeding biology (sequential hermaphrodites) and symbiotic
            mutualism with sea anemones have attracted much scientific interest.
            Moreover, there are some curious geographic-based phenotypes which
            warrant investigation. Leveraging on the advancement in Nanopore
            long read technology, we report the first hybrid assembly of the
            clown anemonefish (Amphiprion ocellaris) genome utilizing Illumina
            and Nanopore reads, further demonstrating the substantial impact of
            modest long read sequencing data sets on improving genome assembly
            statistics. We generated 43 Gb of short Illumina reads and 9 Gb of
            long Nanopore reads representing an approximate genome coverage of
            54x and 11, respectively, based on the range of estimated
            k-mer-predicted genome sizes of between 791 to 967 Mbp. The final
            assembled genome size is contained in 6,404 scaffolds with an
            accumulated length of 880 Mb (96.3% BUSCO-calculated genome
            completeness). Compared to the Illumina-only assembly, the hybrid
            approach generated 94% fewer scaffolds with 18-fold increase in N50
            length (401 kb) and increased the genome completeness by an
            additional 16%. A total of 27,240 high quality protein-coding genes
            were predicted from the clown anemonefish, 26,211 (96%) of which
            were annotated functionally with information from either sequence
            homology or protein signature searches. We present the first genome
            of any anemonefish and demonstrate the value of low coverage (~11x)
            long Nanopore reads sequencing in improving both genome assembly
            contiguity and completeness. The near-complete assembly of the A.
            ocellaris genome will be an invaluable molecular resource for
            supporting a range of genetic, genomic and phylogenetic studies
            specifically for clownfish and more generally for other related fish
            species of the family Pomacentridae.
          </td>
        </tr>
        <tr>
          <td>======================</td>
          <td>======================</td>
        </tr>
        <tr>
          <td>Links</td>
          <td>&nbsp;</td>
        </tr>
        <!--  If no Links then do not display this block, ensure the ==== block above is also not displayed -->
        <tr>
          <td>----------------------</td>
          <td>----------------------</td>
        </tr>
        <tr>
          <td>Related manuscript</td>
          <td>
            doi:<a
              href="http://doi.org/10.1186/s13742-016-0141-6"
              target="_blank"
              rel="noopener"
              >10.1186/s13742-016-0141-6</a
            >
          </td>
        </tr>
        <tr>
          <td>----------------------</td>
          <td>----------------------</td>
        </tr>
        <tr>
          <td>Resourced_Accession</td>
          <td>
            GENBANK:
            <a
              href="http://www.ebi.ac.uk/ena/data/view/NXFZ00000000"
              target="_blank"
              rel="noopener"
              >NXFZ00000000</a
            >
          </td>
        </tr>
        <tr>
          <td>Resourced_Accession</td>
          <td>
            BioProject:
            <a
              href="https://www.ebi.ac.uk/ena/data/view/PRJNA407816"
              target="_blank"
              rel="noopener"
              >PRJNA407816</a
            >
          </td>
        </tr>
        <tr>
          <td>----------------------</td>
          <td>----------------------</td>
        </tr>
        <tr>
          <td>Referenced_Accession</td>
          <td>
            BioProject:
            <a
              href="https://www.ebi.ac.uk/ena/data/view/PRJNA374650"
              target="_blank"
              rel="noopener"
              >PRJNA374650</a
            >
          </td>
        </tr>
        <tr>
          <td>----------------------</td>
          <td>----------------------</td>
        </tr>
        <tr>
          <td>Protocols.io</td>
          <td>
            <a
              href="https://dx.doi.org/10.17504/protocols.io.grkbv4w"
              target="blank"
              >https://dx.doi.org/10.17504/protocols.io.grkbv4w</a
            >
          </td>
        </tr>
        <!--  protocols.io links need to be converted from widgets to DOI's -->
        <tr>
          <td>GitHub_Archive</td>
          <td>
            <a href="https://github.com/htpmod/HTPmod" target="blank"
              >https://github.com/htpmod/HTPmod</a
            >
          </td>
        </tr>
        <tr>
          <td>JBrowse</td>
          <td>
            <a
              href="http://penguin.genomics.cn/jbrowse/index.html?data=100240"
              target="blank"
              >http://penguin.genomics.cn/jbrowse/index.html?data=100240</a
            >
          </td>
        </tr>
        <tr>
          <td>Additional_Info_Link</td>
          <td>
            <a href="http://toolkit.bcblab.com" target="blank"
              >http://toolkit.bcblab.com</a
            >
          </td>
        </tr>
        <tr>
          <td>======================</td>
          <td>======================</td>
        </tr>
        <tr>
          <td>Funding</td>
          <td>&nbsp;</td>
        </tr>
        <!--  If no Funding info then do not display this block, ensure the ==== block above is also not displayed -->
        <tr>
          <td>----------------------</td>
          <td>----------------------</td>
        </tr>
        <tr>
          <td>Funding_body</td>
          <td>Monash University</td>
        </tr>
        <tr>
          <td>Funding_comment</td>
          <td>Monash University Malaysia</td>
        </tr>
        <tr>
          <td>Funding_Awardee</td>
          <td>CM Austin and HM Gan</td>
        </tr>
        <tr>
          <td>----------------------</td>
          <td>----------------------</td>
        </tr>
        <tr>
          <td>Funding_body</td>
          <td>Deakin University</td>
        </tr>
        <tr>
          <td>Funding_Awardee</td>
          <td>CM Austin and HM Gan</td>
        </tr>
        <tr>
          <td>----------------------</td>
          <td>----------------------</td>
        </tr>
        <tr>
          <td>more_funding</td>
          <td>
            repeat blocks of funding stuff for each different funding details
          </td>
        </tr>
        <tr>
          <td>======================</td>
          <td>======================</td>
        </tr>
        <tr>
          <td>Samples</td>
          <td>&nbsp;</td>
        </tr>
        <!--  If no samples then do not display this block, ensure the ==== block above is also not displayed -->
        <tr>
          <td>----------------------</td>
          <td>----------------------</td>
        </tr>
        <tr>
          <td>Sample ID</td>
          <td>
            <a
              href="https://www.ebi.ac.uk/ena/data/view/SAMN06329912"
              target="_blank"
              rel="noopener"
              >SAMN06329912</a
            >
          </td>
        </tr>
        <tr>
          <td>Taxonomic ID</td>
          <td>
            <a
              href="https://www.ebi.ac.uk/ena/data/view/Taxon:80972"
              target="_blank"
              rel="noopener"
              >80972</a
            >
          </td>
        </tr>
        <tr>
          <td>Common Name</td>
          <td>clown anemonefish</td>
        </tr>
        <tr>
          <td>Genbank Name</td>
          <td>&nbsp;</td>
        </tr>
        <tr>
          <td>Scientific Name</td>
          <td>Amphiprion ocellaris</td>
        </tr>
        <tr>
          <td>Sample Attribute</td>
          <td>Description:Male Amphiprion ocellaris</td>
        </tr>
        <tr>
          <td>Sample Attribute</td>
          <td>Sex:male</td>
        </tr>
        <tr>
          <td>Sample Attribute</td>
          <td>Age:1 year</td>
        </tr>
        <tr>
          <td>Sample Attribute</td>
          <td>Life stage:not provided</td>
        </tr>
        <tr>
          <td>Sample Attribute</td>
          <td>
            Tissue type:heart, brain, pituitary, muscle, head kidney, kidney,
            liver, gut, spleen, gill and testis
          </td>
        </tr>
        <tr>
          <td>Sample Attribute</td>
          <td>
            Alternative accession-BioProject:<a
              href="https://www.ebi.ac.uk/ena/data/view/PRJNA374650"
              target="_blank"
              rel="noopener"
              >PRJNA374650</a
            >
          </td>
        </tr>
        <tr>
          <td>----------------------</td>
          <td>----------------------</td>
        </tr>
        <tr>
          <td>Sample_ID</td>
          <td>
            <a
              href="https://www.ebi.ac.uk/ena/data/view/SAMN06329913"
              target="_blank"
              rel="noopener"
              >SAMN06329913</a
            >
          </td>
        </tr>
        <tr>
          <td>Taxonomic_ID</td>
          <td>
            <a
              href="https://www.ebi.ac.uk/ena/data/view/Taxon:80972"
              target="_blank"
              rel="noopener"
              >80972</a
            >
          </td>
        </tr>
        <tr>
          <td>Common_Name</td>
          <td>clown anemonefish</td>
        </tr>
        <tr>
          <td>Genbank_Name</td>
          <td>&nbsp;</td>
        </tr>
        <tr>
          <td>Scientific_Name</td>
          <td>Amphiprion ocellaris</td>
        </tr>
        <tr>
          <td>Sample_Attribute</td>
          <td>Description:Female Amphiprion ocellaris</td>
        </tr>
        <tr>
          <td>Sample_Attribute</td>
          <td>Sex:Female</td>
        </tr>
        <tr>
          <td>Sample_Attribute</td>
          <td>Age:1 year</td>
        </tr>
        <tr>
          <td>Sample_Attribute</td>
          <td>Life stage:not provided</td>
        </tr>
        <tr>
          <td>Sample_Attribute</td>
          <td>
            Tissue type:heart, brain, pituitary, muscle, head kidney, kidney,
            liver, gut, spleen, gill and ovary
          </td>
        </tr>
        <tr>
          <td>Sample_Attribute</td>
          <td>
            Alternative accession-BioProject:<a
              href="https://www.ebi.ac.uk/ena/data/view/PRJNA374650"
              target="_blank"
              rel="noopener"
              >PRJNA374650</a
            >
          </td>
        </tr>
        <tr>
          <td>----------------------</td>
          <td>----------------------</td>
        </tr>
        <tr>
          <td>more_samples</td>
          <td>repeat blocks of sample stuff for each sample</td>
        </tr>
        <tr>
          <td>======================</td>
          <td>======================</td>
        </tr>
        <tr>
          <td>Files</td>
          <td>&nbsp;</td>
        </tr>
        <!--  If no Files then do not display this block, ensure the ==== block above is also not displayed -->
        <tr>
          <td>----------------------</td>
          <td>----------------------</td>
        </tr>
        <tr>
          <td>File_name</td>
          <td>Illumina_assembly.fasta.gz</td>
        </tr>
        <tr>
          <td>File_Location</td>
          <td>
            <a
              href="ftp://penguin.genomics.cn/pub/10.5524/100001_101000/100397/Illumina_assembly.fasta.gz"
              target="_blank"
              rel="noopener"
              >ftp://penguin.genomics.cn/pub/10.5524/100001_101000/100397/Illumina_assembly.fasta.gz</a
            >
          </td>
        </tr>
        <tr>
          <td>File_Description</td>
          <td>
            De novo assembly using Illumina only short reads generated from
            HiSeq and MiSeq (&gt; 500 bp)
          </td>
        </tr>
        <tr>
          <td>File_Type</td>
          <td>annotation</td>
        </tr>
        <tr>
          <td>File_Format</td>
          <td>unknown</td>
        </tr>
        <tr>
          <td>File_Size</td>
          <td>640.17 MB</td>
        </tr>
        <tr>
          <td>File_Release_date</td>
          <td>2017-12-19</td>
        </tr>
        <tr>
          <td>File_Attribute</td>
          <td>&nbsp;</td>
        </tr>
        <!--  If no File_Attributes then do not display this row -->
        <tr>
          <td>File_Sample_ID</td>
          <td>&nbsp;</td>
        </tr>
        <!--  If no File_Sample_IDs then do not display this row -->
        <tr>
          <td>----------------------</td>
          <td>----------------------</td>
        </tr>
        <tr>
          <td>File_name</td>
          <td>Illumina_assembly.fasta.gz</td>
        </tr>
        <tr>
          <td>File_Location</td>
          <td>
            <a
              href="ftp://penguin.genomics.cn/pub/10.5524/100001_101000/100397/Illumina_assembly.fasta.gz"
              target="_blank"
              rel="noopener"
              >ftp://penguin.genomics.cn/pub/10.5524/100001_101000/100397/Illumina_assembly.fasta.gz</a
            >
          </td>
        </tr>
        <tr>
          <td>File_Description</td>
          <td>
            De novo assembly using Illumina only short reads generated from
            HiSeq and MiSeq (&gt; 500 bp)
          </td>
        </tr>
        <tr>
          <td>File_Type</td>
          <td>annotation</td>
        </tr>
        <tr>
          <td>File_Format</td>
          <td>unknown</td>
        </tr>
        <tr>
          <td>File_Size</td>
          <td>640.17 MB</td>
        </tr>
        <tr>
          <td>File_Release_date</td>
          <td>2017-12-19</td>
        </tr>
        <tr>
          <td>File_Attribute</td>
          <td>&nbsp;</td>
        </tr>
        <!--  If no File_Attributes then do not display this row -->
        <tr>
          <td>File_Sample_ID</td>
          <td>&nbsp;</td>
        </tr>
        <!--  If no File_Sample_IDs then do not display this row -->
        <tr>
          <td>----------------------</td>
          <td>----------------------</td>
        </tr>
        <tr>
          <td>more_files</td>
          <td>repeat block of rows above for each file in dataset</td>
        </tr>
        <tr>
          <td>======================</td>
          <td>======================</td>
        </tr>
        <tr>
          <td>License</td>
          <td>
            All files and data are distributed under the Creative Commons
            Attribution-CC0 License unless specifically stated otherwise, see
            <a href="http://gigadb.org/site/term" target="_blank" rel="noopener"
              >http://gigadb.org/site/term</a
            >
            for more details.
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</html>
